.github/workflows/ci.yml

```diff
@@ -82,37 +80,50 @@ jobs:
         phpize
         ./configure
         make
-        make test
-
-
-  #   linux-arm:
-  #     name: ARM (${{ matrix.arch }})
-  #     runs-on: ubuntu-latest
-  #     strategy:
-  #       matrix:
-  #         arch: [armv6, armv7, arm64]
-  #     steps:
-  #     - name: Build ${{ env.PACKAGE_NAME }}
-  #       run: |
-  #         python3 -c "from urllib.request import urlretrieve; urlretrieve('${{ env.BUILDER_HOST }}/${{ env.BUILDER_SOURCE }}/${{ env.BUILDER_VERSION }}/builder.pyz?run=${{ env.RUN }}', 'builder')"
-  #         chmod a+x builder
-  #         ./builder build -p ${{ env.PACKAGE_NAME }} --target=linux-${{ matrix.arch }} --spec=downstream
-
-  # windows-vc16:
-  #   runs-on: windows-latest
-  #   strategy:
-  #     matrix:
-  #       arch: [x64]
-  #   steps:
-  #   - uses: ilammy/msvc-dev-cmd@v1
-  #     with:
-  #       arch: ${{ matrix.arch }}
-  #       uwp: false
-  #       spectre: true
-  #   - name: Build ${{ env.PACKAGE_NAME }} + consumers
-  #     run: |
-  #       python -c "from urllib.request import urlretrieve; urlretrieve('${{ env.BUILDER_HOST }}/${{ env.BUILDER_SOURCE }}/${{ env.BUILDER_VERSION }}/builder.pyz?run=${{ env.RUN }}', 'builder.pyz')"
-  #       python builder.pyz build -p ${{ env.PACKAGE_NAME }} --spec=downstream
+        ./dev-scripts/run_tests
+
+    # linux-arm:
+    #   name: ARM (${{ matrix.arch }})
+    #   runs-on: ubuntu-latest
+    #   strategy:
+    #     matrix:
+    #       arch: [armv6, armv7, arm64]
+    #   steps:
+    #   - name: Build ${{ env.PACKAGE_NAME }}
+    #     run: |
+    #       python3 -c "from urllib.request import urlretrieve; urlretrieve('${{ env.BUILDER_HOST }}/${{ env.BUILDER_SOURCE }}/${{ env.BUILDER_VERSION }}/builder.pyz?run=${{ env.RUN }}', 'builder')"
+    #       chmod a+x builder
+    #       ./builder build -p ${{ env.PACKAGE_NAME }} --target=linux-${{ matrix.arch }} --spec=downstream
+
+  windows-vc16:
+    runs-on: windows-2019
+    defaults:
+      run:
+        shell: cmd # use CMD instead of powershell to catch error from bat script
+    strategy:
+      matrix:
+        arch: [x64]
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: recursive
+    - id: setup-php-sdk
+      uses: cmb69/setup-php-sdk@v0.6
+      with:
+        version: '8.0'
+        arch: x64
+        ts: ts
```

Comment:
Confused: Why do we need a thread safe version? Should we check this for nts as well? Should we have a matrix for PHP versions?

Reply:
We have a matrix of php version on linux, it's kind covered some of those versioning problems. 

So, we currently don't have CI for other versions of php for mac and windows. We can introduce them if needed in the future.

Reply:
I think it would be better to have a matrix or at least a TODO, so that we can catch build failures beforehand in the future instead of someone creating an issue saying that it is not working.